wandb/integration/keras/callbacks/model_checkpoint.py

```diff
@@ -4,6 +4,8 @@
 import sys
 from typing import Any, Dict, List, Optional, Union
 
+import tensorflow as tf  # type: ignore
+from pkg_resources import parse_version
```

Comment:
Please move this import to the method def where it's used. It's notoriously slow so we removed module-level imports of it everywhere.

Reply:
qq: Is importing slow or using the `parse_version` function slow?

@dmitryduev `parse_version` is used `on_train_batch_end` and `on_epoch_end`. If we were to move the import there,  it would slow down the callback which is not ideal.